python/paddle/fluid/parallel_executor.py

```diff
@@ -22,10 +22,12 @@
 
 class ParallelExecutor(object):
     def __init__(self,
-                 loss_name,
                  use_cuda,
+                 loss_name=None,
+                 main_program=None,
                  num_threads=None,
-                 allow_op_delay=False):
+                 allow_op_delay=False,
+                 share_vars_from=None):
```

Comment:
need comments and small examples for constructor

Reply:
Done.